.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,54 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.231.3/containers/python-3
+{
+	"name": "Python 3",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": "..",
+		"args": { 
+			// Update 'VARIANT' to pick a Python version: 3, 3.10, 3.9, 3.8, 3.7, 3.6
+			// Append -bullseye or -buster to pin to an OS version.
+			// Use -bullseye variants on local on arm64/Apple Silicon.
```

Comment:
How does that work exactly for non-ARM?

Reply:
This is fully transparent. The ARM enabled docker images are multi-platform and docker will automatically pull the correct platform image. Some of the older images are x86 only. Those work on M1 Macs but only in slow emulation mode.